### PR TITLE
Add winston

### PR DESCRIPTION
**Pull request by kelian200**
# Description

Added Winston with the current config needs. Only works to output to console but can be changed to output to file if needed.

Winston added to dependencies

Fixes # (issue/ticket number)

Adding Winston for logging

## Type of change

Setup

# Tests:

- I have checked error, warn, and info calls from a separate file in the same dir and it all works as expected.

### CODE REVIEW

File: logging/logger.ts
Comment (r-zhao42) [inline logging/logger.ts:1]: Why is there a .ts and .js file for logger? is it a winston requirement?

Comment (kelian200) [reply logging/logger.ts:1]: oops i forgot to remove the compiled versions.....

**Review by r-zhao42 — APPROVED** [review]

**Comment by r-zhao42** [comment]
Looks like there are some conflicts that need to be resolved before it can be merged in

**Comment by kelian200** [comment]
> Looks like there are some conflicts that need to be resolved before it can be merged in

Can you let me know where because GitHub says there are no conflicts with the base branch

**Comment by r-zhao42** [comment]
hmm, github just says "This branch cannot be rebased due to conflicts" so i'm not sure

**Comment by r-zhao42** [comment]
if you can't see the conflicts, maybe you can merge it?

**Pull request merged**